src/components/BookDetail/BookDetail.jsx

```diff
@@ -36,12 +38,13 @@ function BookDetail() {
                     setBookPageCount(bookData.data.page_count)
                     setBookGenre(bookData.data.genre.name)
                     setBookBlurb(bookData.data.blurb)
+                    setClaimedByName(bookData.data.claimed_by_name)
                 } else {
                     // Display an error message
                     setError(true)
                 }
             })
-    }, [id])
+    }, [id, claimedByName])
```

Comment:
Why does the `useEffect` need claimedByName as a dependency?

Reply:
we've removed claimedbyName from there now.